### ISSUE TITLE
Group blend mode is not set

### ISSUE THREAD

**Issue by sasensi** [Agamnentzar/ag-psd#56]
Given this input data:
```
{
  width: 573,
  height: 351,
  children: [
    {
      children: [
        {
          vectorFill: { type: 'color', color: { r: 255, g: 0, b: 0 } },
          vectorMask: { paths: [{ knots: [{ points: [100, 128, 100, 128, 100, 128] }, { points: [300, 128, 300, 128, 300, 128] }, { points: [300, 228, 300, 228, 300, 228] }, { points: [100, 228, 100, 228, 100, 228] }] }] },
        },
      ],
    },
    {
      name: 'Target group',
      blendMode: 'multiply',
      children: [
        {
          vectorFill: { type: 'color', color: { r: 0, g: 0, b: 255 } },
          vectorMask: {
            paths: [{
              knots: [{ points: [200, 178, 200, 178, 200, 178] }, { points: [400, 178, 400, 178, 400, 178] }, { points: [400, 278, 400, 278, 400, 278] }, { points: [200, 278, 200, 278, 200, 278] }],
            }],
          },
        },
      ],
    },
  ],
}
```

Here is the generated file:
![Capture d’écran 2021-03-24 184542](https://user-images.githubusercontent.com/11247504/112359063-2f9ffe00-8cd1-11eb-90a6-12b7392f6b40.png)

The "Target group" layer blend mode should be set to "multiply" and produce this output:
![Capture d’écran 2021-03-24 184555](https://user-images.githubusercontent.com/11247504/112359149-434b6480-8cd1-11eb-908a-4038397f7463.png)

**Comment by Agamnentzar** [comment]
Should now be fixed in version `11.6.1`

**Comment by sasensi** [comment]
Perfect, I confirm that this is fixed.
Thanks a lot.
